### PR TITLE
Updated wjoy (0.7.1)

### PR DESCRIPTION
**Pull request by tsparber**
Refs https://github.com/alxn1/wjoy/issues/88

But I'm not sure if yandex supports hotlinking.

Tick #17075

### CODE REVIEW

File: Casks/wjoy.rb [hunk 1]
```diff
@@ -2,9 +2,12 @@
   version '0.7.1'
   sha256 '98a9f825634b09b901ad979a6cdda241a04a1cbd7dcd14b4b17b0bebe3f40ee1'
 
-  url "https://wjoy.googlecode.com/files/wjoy%20#{version}.dmg"
+  # yandex.com is the official download host per the vendor homepage
+  url "https://downloader.disk.yandex.com/disk/ed9d7c0be605951db5059165769cb6e3963e75ceeef18c4b441f505737569ae5/569d9c29/okcwpAt_Vv--JfKp9linRPvNy5QqC4Q2EjZSlY51gfn7BLPYSf3EMCQlU38NIe9B-wDC6DBBUGEWk0fPyFw8Cg%3D%3D?uid=0&filename=wjoy%20#{version}.dmg&disposition=attachment&hash=cU4dkqvlum9GDNAywQSQiZQNxAC1DD9FZaZTwhLKGx4%3D&limit=0&content_type=application%2Fx-apple-diskimage&fsize=1226259&hid=9068c6da8da41bed302e5267ab842dbf&media_type=compressed&tknv=v2"
```
Comment (tsparber) [inline Casks/wjoy.rb]: The homepage links to https://yadi.sk/d/Fj6PYXpKiMPwc which then provides the downloader.disk.yandex.com link. So the comment is not entirely correct. But yadi.sk seems to be an "official" url shortener for yandex disk.

File: Casks/wjoy.rb [hunk 2]
```diff
@@ -2,9 +2,12 @@
...
+  appcast 'https://raw.githubusercontent.com/alxn1/wjoy/master/README.md',
```
Comment (vitorgalvao) [inline Casks/wjoy.rb]: Unusual choice for the `appcast`, but I guess it works.

File: Casks/wjoy.rb (hunk 1 above)
Comment (vitorgalvao) [inline Casks/wjoy.rb]: Looks like it won’t work. Just tried downloading it and was unsuccessful. Can you do it?

**Comment by vitorgalvao** [comment]
This needs the `discontinued` caveat.

**Comment by tsparber** [comment]
amended

**Comment by tsparber** [comment]
Fixed, finally :smiley:

**Pull request merged**